kuryr_kubernetes/controller/drivers/lbaasv2.py

```diff
@@ -403,6 +409,22 @@ def _extend_lb_security_group_rules(self, loadbalancer, listener):
                               'to enable routes for listener %s.',
                               listener.name)
 
+    def _get_remote_ip_from_sg(sg):
+        neutron = clients.get_neutron_client()
+        sg_rules = neutron.list_security_group_rules(
+            security_group_id=sg,
+            direction='ingress',
+            description='Kuryr Namespace SG rule')
+
+        if len(sg_rules['security_group_rules']) != 1:
+            LOG.warning("There are more than one Kuryr SG rules associated "
+                        "to the namespace. This means manual addition and "
+                        "may break the isolation and lead to side effects.")
+        # Only 1 should exits. Returning the first one
+        if sg_rules['security_group_rules'][0].get('remote_ip_prefix'):
```

Comment:
Why not just` return sg_rules['security_group_rules'][0].get('remote_ip_prefix')`?

Reply:
right!